src/components/dialogs/contingency-list/filter-based/equipment-types-by-filters.tsx

```diff
@@ -0,0 +1,167 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import {
+    CONTINGENCY_LIST_EQUIPMENTS,
+    ContingencyListEquipment,
+    OverflowableTableCell,
+    OverflowableTableCellWithCheckbox,
+} from '@gridsuite/commons-ui';
+import { Grid, Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from '@mui/material';
+import { useFormContext, useWatch } from 'react-hook-form';
+import { useCallback, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { ContingencyFieldConstants, FilterElement, FilterSubEquipments } from '../../../../utils/contingency-list.type';
+
+interface EquipmentTypesByFiltersProps {
+    substationAndVLFilters: ReadonlyArray<FilterElement>;
+    setIsDataOutdated: (value: boolean) => void;
+}
+
+export function EquipmentTypesByFilters({
```

Comment:
```suggestion
function EquipmentTypesByFilters({
```
use export default at the bottom